libraries/Microsoft.Bot.Builder.AI.LuisV3/LuisRecognizer.cs

```diff
@@ -0,0 +1,445 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder.TraceExtensions;
+using Microsoft.Bot.Schema;
+using Microsoft.Rest;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Bot.Builder.AI.Luis
+{
+    /// <inheritdoc />
+    /// <summary>
+    /// A LUIS based implementation of <see cref="ITelemetryRecognizer"/> for the V3 endpoint.
+    /// </summary>
+    public class LuisRecognizer : ITelemetryRecognizer
+    {
+        /// <summary>
+        /// The value type for a LUIS trace activity.
+        /// </summary>
+        public const string LuisTraceType = "https://www.luis.ai/schemas/trace";
+
+        /// <summary>
+        /// The context label for a LUIS trace activity.
+        /// </summary>
+        public const string LuisTraceLabel = "LuisV3 Trace";
+
+        private readonly LuisApplication _application;
+        private readonly LuisPredictionOptions _predictionOptions;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="LuisRecognizer"/> class.
+        /// </summary>
+        /// <param name="application">The LUIS application to use to recognize text.</param>
+        /// <param name="recognizerOptions">(Optional) Options for the created recognizer.</param>
+        /// <param name="predictionOptions">(Optional) The default LUIS prediction options to use.</param>
+        /// <param name="clientHandler">(Optional) Custom handler for LUIS API calls to allow mocking.</param>
```

Comment:
There's no parameter with the name `clientHandler`.

Reply:
Nice catch--that got moved into the recognizer options so I removed this.